lib/timely/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Timely
-  VERSION = "0.5.0"
```

Comment:
Normally we bump the version when doing the release, not sure it really matters but may be confusing if we actually had multiple people working on these things at a time

Reply:
Yeah I got ahead of myself here.